pkg/controller/containerjfr/containerjfr_controller.go

```diff
@@ -72,8 +76,9 @@ var _ reconcile.Reconciler = &ReconcileContainerJFR{}
 type ReconcileContainerJFR struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client client.Client
-	scheme *runtime.Scheme
+	client      client.Client
+	routeClient routeClient.RouteV1Client
```

Comment:
I think we might be able to use the existing client by adding Route to our Scheme, but this is fine for now.

Reply:
Good to know. I figured there must be a better way to do it, but I couldn't figure it out.

Reply:
I think I have a rough idea how to do it. I'll play around with it.